src/sentry/conf/server.py

```diff
@@ -1484,6 +1484,7 @@ def create_partitioned_queues(name):
     },
     "zookeeper": {
         "image": "confluentinc/cp-zookeeper:5.1.2",
+        "ports": {"2181/tcp": 2181},
```

Comment:
🤔 This is just purely for annotative purposes which could be improved, but thanks for making it consistent.